<launch>
	<node pkg="seabee3_driver_base" type="seabee3_driver_base" name="seabee3_driver_base" output="screen" >
		<param name="usb_index" type="int" value="0" />
		<param name="shooter/trigger_time" type="int" value="50" />
		<param name="shooter/trigger_value" type="int" value="80" />
		
		<param name="dropper1/trigger_time" type="int" value="125" />
		<param name="dropper1/trigger_value" type="int" value="75" />
		
		<param name="dropper2/trigger_time" type="int" value="125" />
		<param name="dropper2/trigger_value" type="int" value="55" />
		<param name="min_motor_value" type="int" value="15" />
		
		<!-- ========== Motor/Behavior Tuning ========== -->
		
		<param name="speed_m1_dir" type="double" value="-1.0" />
		<param name="speed_m2_dir" type="double" value="-1.0" />
		
		<param name="strafe_m1_dir" type="double" value="-1.0" />
		<param name="strafe_m2_dir" type="double" value="-1.0" />
		
		<param name="depth_m1_dir" type="double" value="-1.0" />
		<param name="depth_m2_dir" type="double" value="-1.0" />
	</node>
</launch>
